src/ports/linux/sampleapp_main.c

```diff
@@ -365,19 +365,19 @@ int main (int argc, char * argv[])
 
    /* Prepare stack config with IP address, gateway, station name etc */
    app_adjust_stack_configuration (&pnet_default_cfg);
-   app_copy_ip_to_struct (&pnet_default_cfg.ip_addr, ip);
-   app_copy_ip_to_struct (&pnet_default_cfg.ip_gateway, gateway);
-   app_copy_ip_to_struct (&pnet_default_cfg.ip_mask, netmask);
+   app_copy_ip_to_struct (&pnet_default_cfg.if_cfg.ip_cfg.ip_addr, ip);
```

Comment:
Maybe I missed it, but don't we need to put in `arguments.eth_interface` into two places in the config now that it not is part of `pnet_init()`?

Reply:
You are correct. A bit worrying that I did miss this and the tests I ran passed. 
Done